### PR TITLE
Mussel 0.1.3

### PR DESCRIPTION
**Pull request by renatogamboa**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/mussel/0.1.3/step.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  git: https://github.com/UrbanCompass/bitrise-step-mussel.git
+  commit: d5f477a83ecb5f6b3068cea92fa0c4dc3b70b2d4
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
```
Comment (koral--) [inline steps/mussel/0.1.3/step.yml:38]: This file does not seem to do anything apart from echo.

File: steps/mussel/0.1.3/step.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+source_code_url: https://github.com/UrbanCompass/bitrise-step-mussel
+support_url: https://github.com/UrbanCompass/bitrise-step-mussel/issues
+published_at: 2021-07-14T17:25:56.747095-04:00
+source:
+  git: https://github.com/UrbanCompass/bitrise-step-mussel.git
+  commit: d5f477a83ecb5f6b3068cea92fa0c4dc3b70b2d4
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
```
Comment (koral--) [inline steps/mussel/0.1.3/step.yml:34]: project_type_tags limiting targets should be there. Eg. android does not seem to make sense here.

File: steps/mussel/0.1.3/step.yml (hunk 1 above)
Comment (renatogamboa) [reply steps/mussel/0.1.3/step.yml:38]: Hi! Thanks for reviewing :) Will remove `toolkit` here and this file.

File: steps/mussel/0.1.3/step.yml (hunk 2 above)
Comment (renatogamboa) [reply steps/mussel/0.1.3/step.yml:34]: Thank you for reviewing @koral-- , greatly appreciate it! Going to close this PR and open a new one with the fixes. Adding `project_type_tags` for `ios`

**Pull request closed without merging**